### PR TITLE
Refs [TASK][41453] Create UI GetStartedFragment

### PR DESCRIPTION
**Pull request by vinhmone**
## Related Tickets
- [#Task 41453: Create UI GetStartedFragment](https://edu-redmine.sun-asterisk.vn/issues/41453)
## WHAT
- 
## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/layout/fragment_get_started.xml [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+    </data>
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@android:color/white"
+        tools:context=".ui.getstarted.GetStartedFragment">
+
+        <ImageView
+            android:id="@+id/image_landscape"
```
Comment (thonc-2135) [inline app/src/main/res/layout/fragment_get_started.xml:20]: convention h để gạch chân à, a tưởng trc là imageLangscape nhỉ

File: app/src/main/res/layout/fragment_get_started.xml [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+        <TextView
+            android:id="@+id/text_title"
+            style="@style/TextViewDefault"
+            android:layout_width="0dp"
+            android:layout_marginStart="@dimen/dp_10"
+            android:ellipsize="end"
+            android:maxLines="1"
+            android:textColor="@android:color/white"
+            android:textSize="@dimen/sp_20"
+            app:layout_constraintBottom_toBottomOf="@+id/view_tool_bar"
```
Comment (thonc-2135) [inline app/src/main/res/layout/fragment_get_started.xml]: sửa lại hết nhé
```suggestion
            app:layout_constraintBottom_toBottomOf="@id/view_tool_bar"
```

File: app/src/main/res/layout/fragment_get_started.xml (hunk 1 above)
Comment (vinhmone) [reply app/src/main/res/layout/fragment_get_started.xml:20]: trước a Đạt bảo e là hồi chưa dùng viewbinding thì phải để kiểu imageLandscape, còn giờ dùng viewbinding thì để kiểu gạch chân cũng được. Với e thấy file coding_convention của công ty cũng để kiểu image_, text_ ý ạ @@

Comment (tuannm-1276) [reply app/src/main/res/layout/fragment_get_started.xml:20]: uki em. cứ triển theo convention của file cty nhé
https://github.com/framgia/coding-standards/blob/master/eng/android/coding_convention_android_kotlin.md

**Pull request merged**